packages/claude-hooks/src/speech/providers/provider-factory.ts

```
@@ -0,0 +1,223 @@
+/**
+ * Factory for creating TTS providers
+ */
+export class TTSProviderFactory {
+  private static providers = new Map<string, ProviderConstructor>()
+
+  /**
+   * Register a provider
+   */
+  static registerProvider(
+    name: string,
+    constructor: ProviderConstructor,
```

Biome (2.1.2)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)